otelx/otel_test.go

```diff
@@ -0,0 +1,53 @@
+package otelx
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"testing"
+
+	"github.com/ory/x/logrusx"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/otel/attribute"
+)
+
+func TestJaegerTracer(t *testing.T) {
+	host := "127.0.0.1"
+	port := "6831"
+	done := make(chan struct{})
+	go func(addr string) {
+		t.Log("Starting test UDP server for Jaeger spans")
+		udpAddr, _ := net.ResolveUDPAddr("udp", addr)
+		srv, _ := net.ListenUDP("udp", udpAddr)
```

Comment:
```suggestion
		udpAddr, err := net.ResolveUDPAddr("udp", addr)
		require.NoErr(t, err)
		srv, err := net.ListenUDP("udp", udpAddr)
		require.NoErr(t, err)
```

Reply:
If you ignore those errors, it is literally impossible to figure out why the test is failing.

Reply:
Ah wait, you can't use `require` in other goroutines. Use an errgroup instead and return the error.